### PR TITLE
render: complete config

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

**Comment by sttts** [comment]
/hold

until smoke tests are green.

File: manifests/bootkube/config/bootstrap-config-overrides.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - "/etc/kubernetes/cloud/config"
+  root-ca-file:
+  - "/etc/kubernetes/secrets/root-ca.crt"
+  service-account-private-key-file:
+  - "/etc/kubernetes/secrets/service-account.key"
+  cluster-signing-cert-file:
+  - "/etc/kubernetes/secrets/kube-ca.crt"
+  cluster-signing-key-file:
+  - "/etc/kubernetes/secrets/kube-ca.key"
+  kubeconfig:
```
Comment (deads2k) [inline manifests/bootkube/config/bootstrap-config-overrides.yaml]: turns out that for some reason we have to pass this on the command line in the pod for now.  don't know why.  Probably loading order.

Comment (sttts) [reply manifests/bootkube/config/bootstrap-config-overrides.yaml]: good to know.

File: manifests/bootkube/config/config-overrides.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  certFile: service-signer.crt
+extendedArguments:
+  cloud-config:
+  - "/etc/kubernetes/cloud/config"
+  root-ca-file:
+  - "/etc/kubernetes/secrets/root-ca.crt"
+  service-account-private-key-file:
+  - "/etc/kubernetes/secrets/service-account.key"
+  cluster-signing-cert-file:
+  - "/etc/kubernetes/secrets/kube-ca.crt"
```
Comment (deads2k) [inline manifests/bootkube/config/config-overrides.yaml]: in some followup, I think we should split secrets like we did in the apiserver

File: manifests/v3.11.0/kube-apiserver/defaultconfig.yaml [hunk 3]
```diff
@@ -1,2 +1,39 @@
...
+  - "10.3.0.0/16"
+  use-service-account-credentials:
+  - "true"
+  leader-elect:
+  - "true"
+  leader-elect-retry-period:
+  - "3s"
+  leader-elect-resource-lock:
+  - "configmaps"
+  controllers:
```
Comment (deads2k) [inline manifests/v3.11.0/kube-apiserver/defaultconfig.yaml]: followup to remove this.  We should try to run straight upstream.

**Comment by deads2k** [comment]
if it runs, I'd merge it.

File: manifests/bootkube/config/config-overrides.yaml (hunk 2 above)
Comment (sttts) [reply manifests/bootkube/config/config-overrides.yaml]: rule of thumb right now: follow kube-core to make replacing it easy. After we have done that we can make everything pretty.

**Comment by sttts** [comment]
/hold

File missing.

**Comment by sttts** [comment]
/hold

Manifest must not have conflicting names.

**Pull request merged**